framework/web/Response.php

```diff
@@ -417,7 +417,7 @@ protected function sendContent()
             return;
         }
 
-        set_time_limit(0); // Reset time limit for big files
+        @set_time_limit(0); // Reset time limit for big files
```

Comment:
How can setting unlimited script timeout cause error? Supressing errors/warnings globally is not good idea.